examples/slurm/prologMoneo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+echo $SLURM_JOB_NODELIST > /tmp/jobid_$SLURM_JOB_ID.txt
+
+# The command below pipes stdout/stderr to files, This is necessary for ansible to behave correctly. 
+sudo -u <Your user> -H sh -c "cd  <Moneo Dir> &&  python3 moneo.py -j $SLURM_JOB_NAME.$SLURM_JOB_ID -c /tmp/jobid_$SLURM_JOB_ID.txt  > /tmp/moneolog.out 2>&1"
```

Comment:
Do we use this `/tmp/moneolog.out` at later point? If multiple job ID assignments happen concurrently, will this file get overwritten?

Reply:
Thats a good observation let me address

Reply:
Addressed